packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -78,6 +81,7 @@ const Dashboard = ({
             }
         }
     };
+    let tour_ended = false;
```

Comment:
@farabi-deriv I assume that would be better to use here` React.useRef()` to be sure that we didn’t lose necessary value during rendering.

Reply:
done ✅